python/py-awscli/Portfile

```diff
@@ -5,24 +5,25 @@ PortGroup           python 1.0
 PortGroup           select 1.0
 
 name                py-awscli
-version             1.18.164
+version             1.19.32
 revision            0
 
+categories          python devel
 platforms           darwin
 supported_archs     noarch
 license             Apache-2
-maintainers         nomaintainer
+maintainers         {outlook.de:judaew @judaew} openmaintainer
 
-description         Universal Command Line Environment for AWS.
+description         Universal Command Line Environment for Amazon Web Services.
 long_description    {*}${description}
 
 homepage            https://aws.amazon.com/cli/
 
-checksums           rmd160  9e9812bf12ec6581a8c760b665d26083fdd5916b \
-                    sha256  e64cd0ba919f37a54affa127baf856a832cf3934089baeb1e7a6b776476f8b2f \
-                    size    1320560
+checksums           rmd160  2785a07f4bde1a4dd1bd1c84b15a081a6b3b0a3b \
+                    sha256  760796a154a8b1b3ae008890edf3b961610ef6f9ea19978316be86b575372591 \
+                    size    1398877
 
-python.versions     38
+python.versions     27 36 37 38
```

Comment:
What about also adding python 3.9 here ?

Reply:
Good idea, thank you